docs/changelog.rst

```diff
@@ -11,8 +11,38 @@ The full release history can be viewed `at the GitHub perses releases page <http
 0.10.0 - Release
 ----------------
 
+New command line interface (CLI), many enhancements for the API (specially the ``PointMutationExecutor``) and improved testing.
+
+Bugfixes
+^^^^^^^^
+- Bug in geometry engine when could not generate a successful topology -- fixed by not trying to access it when its ``None``. (`#855 <https://github.com/choderalab/perses/pull/855>`_)
```

Comment:
```suggestion
- Bug in geometry engine when `topology_index_map` is None -- fixed by not trying to access it when its ``None``. (`#855 <https://github.com/choderalab/perses/pull/855>`_)
```

Reply:
Why are there so many underscores? For example there is one after the link to the PR in this line.